emf/loadflow_tool/merge.py

```diff
@@ -73,58 +73,47 @@
-                                'pmd:creationDate': f"{datetime.datetime.utcnow():%Y-%m-%dT%H:%M:%S.%fZ}",
-                                'pmd:timeHorizon': time_horizon,
-                                'pmd:cgmesProfile': 'SV',
-                                'pmd:contentType': 'CGMES',
-                                'pmd:modelPartReference': '',
-                                'pmd:mergingEntity': 'BALTICRSC',
-                                'pmd:mergingArea': area,
-                                'pmd:validFrom': f"{parse_datetime(scenario_date):%Y%m%dT%H%MZ}",
-                                'pmd:modelingAuthoritySet': 'http://www.baltic-rsc.eu/OperationalPlanning',
-                                'pmd:scenarioDate': f"{parse_datetime(scenario_date):%Y-%m-%dT%H:%M:00Z}",
-                                'pmd:modelid': SV_ID,
-                                'pmd:description':
+CGM_meta = {'pmd:fullModel_ID': SV_ID,
+            'pmd:creationDate': f"{datetime.datetime.utcnow():%Y-%m-%dT%H:%M:%S.%fZ}",
+            'pmd:timeHorizon': time_horizon,
+            'pmd:cgmesProfile': 'SV',
+            'pmd:contentType': 'CGMES',
+            'pmd:modelPartReference': '',
+            'pmd:mergingEntity': 'BALTICRSC',
+            'pmd:mergingArea': area,
+            'pmd:validFrom': f"{parse_datetime(scenario_date):%Y%m%dT%H%MZ}",
+            'pmd:modelingAuthoritySet': 'http://www.baltic-rsc.eu/OperationalPlanning',
+            'pmd:scenarioDate': f"{parse_datetime(scenario_date):%Y-%m-%dT%H:%M:00Z}",
+            'pmd:modelid': SV_ID,
+            'pmd:description':
 f"""<MDE>
     <BP>{time_horizon}</BP>
     <TOOL>pypowsybl_{pypowsybl.__version__}</TOOL>
     <RSC>BALTICRSC</RSC>
 </MDE>""",
                                 'pmd:versionNumber': version,
                                 'file_type': "xml"}
-            }
 
 #temp_dir = tempfile.mkdtemp()
-temp_dir = ""
-export_file_path = os.path.join(temp_dir, f"MERGED_SV_{uuid.uuid4()}.zip")
-logger.info(f"Exprting merged model to {export_file_path}")
 
 export_report = pypowsybl.report.Reporter()
-merged_model["NETWORK"].dump(export_file_path,
-                           format="CGMES",
-                           parameters={
-                                "iidm.export.cgmes.modeling-authority-set": CGM_meta['opdm:OPDMObject']['pmd:modelingAuthoritySet'],
-                                "iidm.export.cgmes.base-name": filename_from_metadata(CGM_meta['opdm:OPDMObject']).split("_SV")[0],
-                                "iidm.export.cgmes.profiles": "SV",
-                                "iidm.export.cgmes.naming-strategy": "cgmes",  # identity, cgmes, cgmes-fix-all-invalid-ids
-                                       })
-
+exported_model = export_model(merged_model["NETWORK"])
+logger.info(f"Exporting merged model to {exported_model.name}")
 
 # Load SV data
-sv_data = pandas.read_RDF([export_file_path])
+sv_data = pandas.read_RDF([exported_model.name])
```

Comment:
just exported_model should work, .name gives path, but read_RDF can load in bytesIO directly